packages/fxa-react/storybooks.js

```diff
@@ -29,18 +29,27 @@ const customizeWebpackConfig = ({ config, mode }) => {
   });
   config.resolve.extensions.push('.scss', '.css');
 
+  let assetLoader;
   const assetRule = config.module.rules.find(({ test }) => test.test('.svg'));
   if (assetRule) {
-    const assetLoader = {
+    assetLoader = {
       loader: assetRule.loader,
       options: assetRule.options || assetRule.query,
     };
-    config.module.rules.unshift({
-      test: /\.svg$/,
-      use: ['@svgr/webpack', assetLoader],
-    });
-    config.module.rules = [{ oneOf: config.module.rules }];
+  } else {
+    assetLoader = {
+      loader: resolve(
+        __dirname,
+        '../../node_modules/@storybook/core/node_modules/file-loader/dist/cjs.js'
```

Comment:
It might be worth throwing a little explainer above this. Something like "fxa-react provides its own file-loader, but fxa-settings doesn't, YMMV with additional storybook setups". I can see this being a somewhat fragile setup.

Reply:
Yeah, good call. I will do this in a follow up PR shortly after this merges, I don't want this to go through CI again for a comment.